suite-native/app/src/navigation/RootStackNavigator.tsx

```diff
@@ -56,10 +56,12 @@ export const RootStackNavigator = () => {
                 component={OnboardingStackNavigator}
             />
             <RootStack.Screen name={RootStackRoutes.AppTabs} component={AppTabNavigator} />
-            <RootStack.Screen
-                name={RootStackRoutes.AccountsImport}
-                component={AccountsImportStackNavigator}
-            />
+            <RootStack.Group screenOptions={{ animation: 'slide_from_bottom' }}>
```

Comment:
[Stack.Group](https://reactnavigation.org/docs/group/) should be used if you want to configure multiple screens at once. Since this group has only one screen, Please remove it and pass the `screenOptions` to the screen directly.

Reply:
I just copypasted this as it was already used in the file, wanted to stay consistent. Will fix all the occurrences.